docs/getting-started/what-is-dotnet.md

```diff
@@ -1,6 +1,150 @@
+</td>
+<td>Includes the .NET Framework Class Library, a large, monolithic library with thousands of classes and many thousands of members. Whether or not your application uses individual types and their members (and most applications take advantage of only a small portion of the .NET Framework's functionality), they are always present and accessible. 
+</td>
+</tr>
+<tr>
+<td>Is suitable for a full range of modern applications, including applications that operate on small devices with constrained memory and storage.
+</td>
+<td>Is suitable for developing traditional Windows desktop applications, including Windows Forms (WinForms) and Windows Presentation Foundation (WPF) applications.
+</td>
+</tr>
+<tr>
+<td>Can be used to develop apps that take advantage of a number of technologies, such as ASP .NET Core, for building web applications; Windows Communication Foundation (WCF), for building services, including SOAP; and Workflow Foundation (WF), for building workflows. 
+</td>
+<td>Can be used to develop apps that take advantage of a number of technologies, such as ASP .NET and ASP.NET Web Forms, for building web applications; Windows Communication Foundation (WCF), for building services, including SOAP; and Workflow Foundation (WF), for building workflows. 
+</td>
+</tr>
+<tr>
+<td>Can be app local. That is, the .NET Core implementation that your app relies is tightly bound to your application. This mitigates versioning problems.
+</td>
+<td>Is globally available on a single system. That is, although an app can include a chain installer that installs a particular version of the .NET Framework if the installation routine finds that it is not present, the full .NET Framework can also be installed and is maintained independently of the app. This can create versioning problems, particularly when an app encounters a version of the .NET Framework that it is not expecting or when an app runs on a version of the .NET Framework against which it was not developed.</p>
+Starting with Windows 8, a version of the .NET Framework is installed as an operating system component and is serviced by Windows Update. For information on the .NET Framework versions installed with specific versions of Windows, see [.NET Framework System Requirements](https://msdn.microsoft.com/library/8z6watww.aspx).  
+</td>
+</tr>
+</table>
+
+Although the .NET Framework and .NET Core target different platforms and represent different approaches to application development and deployment, they both conform to the .NET Standard 1.3. This means that they offer a high degree of compatibility and identical behavior between one another. In particular:
```

Comment:
Right now the [corefx doc](https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md) can be linked to.  Unfortunately we don't have any other docs to link to at this time.